testsuite/tests/build_profile/last_profile/test.py

```diff
@@ -0,0 +1,38 @@
+"""
+Check that the last build profile is stored properly, no matter its procedence
+"""
+
+from drivers.alr import run_alr, init_local_crate, alr_manifest
+from drivers.asserts import assert_eq, assert_match
+
+init_local_crate("xxx")
+
+# Check profile is the default one (development) if unspecified
+
+run_alr("build")
+assert_match(".*last_build_profile=DEVELOPMENT.*",
+             run_alr("config").out)
+
+# Check explicit profile in command line
+
+run_alr("build", "--release")
+assert_match(".*last_build_profile=RELEASE.*",
+             run_alr("config").out)
+
+# Check implicit profile when build is indirect
+
+run_alr("run")  # Causes a default build
```

Comment:
I wonder if `alr run` should keep the last build profile.

Reply:
Yep, it's probably more intuitive that way. I'll add that change.

Reply:
But in a follow-up patch